dandi/cli/cmd_validate.py

```diff
@@ -80,6 +86,26 @@ def validate_bids(
     default="none",
 )
 @click.option("--ignore", metavar="REGEX", help="Regex matching error IDs to ignore")
+@click.option(
+    "--match",
+    metavar="REGEX,REGEX,...",
+    help=(
+        "Comma-separated regex patterns used to filter issues in validation results "
+        "by their ID. Only issues with an ID matching at least one of the given "
+        "patterns are included in the eventual result. "
+        "(No pattern should contain a comma.)"
+    ),
+    callback=parse_regexes,
+)
+@click.option(
+    "--include-path",
```

Comment:
- name was a little confusing as I thought "include name in the result"
- we might want to simply (ab)use existing `paths` posarg as it is to define what paths we care about. So even if within bids dataset, bids-validator-deno would run on entire dataset ATM and then we need to filter for those paths
  - we should also warn about that -- that we do run bids-validator on the full dandiset although interested only in some